packages/backend-app-api/README.md

```diff
@@ -0,0 +1,17 @@
+# @backstage/backend-app-api
+
+This package provides the core API used by Backstage backend apps.
```

Comment:
Let's add a fat disclaimer at the top of these that it's all experimental and should not be used